neqo-http3/src/stream_type_reader.rs

```diff
@@ -28,6 +28,7 @@ pub enum NewStreamType {
     WebTransportStream(u64),
     Http,
     Unknown,
+    Grease(u64),
```

Comment:
We don't need to identify GREASE on the way in, so I don't think that we need this.